vllm/_custom_ops.py

```diff
@@ -266,7 +270,16 @@ def awq_gemm(input: torch.Tensor, qweight: torch.Tensor, qzeros: torch.Tensor,
     return torch.ops._C.awq_gemm(input, qweight, qzeros, scales, split_k_iters)
 
 
-# gptq
+def gptq_hpu_gemm(a: torch.Tensor, b_q_weight: torch.Tensor,
```

Comment:
Should it be move to vllm-hpu-extension ops.py?

Reply:
I followed the convention of cuda implementation:

https://github.com/HabanaAI/vllm-fork/blob/2ea889afdac9df62190de14113e00e293d8e3aff/vllm/_custom_ops.py#L283